src/server/api/routes/users/v2/event.js

```diff
@@ -1,20 +1,41 @@
 const { Router } = require('express');
 const {
   walletUserCollection,
-  FieldValue,
+  Timestamp,
 } = require('../../../../modules/firebase');
 const { authenticateV2Login } = require('../../../middleware/auth');
 const { setPrivateCacheHeader } = require('../../../middleware/cache');
 const { publisher, PUBSUB_TOPIC_MISC } = require('../../../../modules/pubsub');
 
 const router = Router();
 
+function isValidTs(ts, lastSeenTs) {
+  // eslint-disable-next-line no-restricted-globals
+  if (isNaN(ts) || ts % 1000 !== 0) {
```

Comment:
`Number.isNaN` is allowed without disabling eslint, do you really need `isNaN` but not `Number.isNaN`?